source/web_integration/pnapi_web_integration.rst

```diff
@@ -68,10 +70,38 @@ Client Side Parameters
     - String
     - Invoca Network ID
 
+
+
+.. list-table::
+  :widths: 11 4 40
+  :header-rows: 1
+  :class: parameters
+
+  * - Required: One of the following
+    -
+    -
+
   * - numberSelector
     - String
     - CSS selector for phone number HTML elements
-  
+      Accepts one or many like so `".invocaNumber, .phoneNumber"`
+
+  * - numberToReplace
+    - String or Hash
+    - Default: `null`
+
+      Specify which number(s) to replace. Can accept one or many like so: `"888-999-1010, 777-888-9999"`.
+      Optionally accepts a hash with the number as a key name, and campaignId as it's value like so:
+      
+      `{ "888-999-1010": 'campaignIdHere' }`
+
+  * - destinationAsId
+    - Bool
+    - Default `false`
+
+      Strips out any special characters and uses found destination phone number's digits as campaign ID.
+      Warning! Requires Invoca campaign with explicit ID for each number. 
```

Comment:
What do you think about bolding this, and possibly putting it on a separate line from the previous sentence?